.github/workflows/deploy-staging.yml

```diff
@@ -22,12 +44,18 @@ jobs:
           role-session-name: CzidGraphQLFederationUpdateStaging
       - name: Install happy
         uses: chanzuckerberg/github-actions/.github/actions/install-happy@main
-      - name: Set version info
+      - name: Set git SHA
         uses: chanzuckerberg/czid-graphql-federation-server/.github/actions/happy-config-set@main
         with:
           app_config_name: CZID_GQL_FED_GIT_SHA
           app_config_value: ${{ github.sha }}
           happy_env: staging
+      - name: Set release version
+        uses: chanzuckerberg/czid-graphql-federation-server/.github/actions/happy-config-set@main
+        with:
+          app_config_name: CZID_GQL_FED_GIT_VERSION
+          app_config_value: ${{ needs.get-release-tag.outputs.release_tag || ''}}
+          happy_env: staging
```

Comment:
We're able to set this in staging now, since we are deploying release tags there first (rather than only to prod).